scout/adapter/mongo/hgnc.py

```diff
@@ -186,7 +186,9 @@ def hgnc_genes_find_one(self, hgnc_symbol, build="37"):
         """
 
         LOG.debug("Find one genes with symbol %s" % hgnc_symbol)
-        return self.hgnc_collection.find_one(filter={"build": build, "aliases": hgnc_symbol})
+        return self.hgnc_collection.find_one(
+            filter={"build": build, "hgnc_symbol": hgnc_symbol, "aliases": hgnc_symbol}
```

Comment:
Hm, aliases should always contain hgnc_symbol. We should probably fix this where it is called. Probably that place should use hgnc_genes instead of hgnc_genes_find_one?